packages/framework/src/Component/DataFixture/AbstractReferenceFixture.php

```diff
@@ -13,6 +16,30 @@ abstract class AbstractReferenceFixture implements FixtureInterface
      */
     protected $persistentReferenceFacade;
 
+    /**
+     * @var bool
+     */
+    protected bool $isLoggerTemporarilyDisabled = false;
```

Comment:
Why is this necessary? The function is called when the instance is created so this value will be always false, right?

Reply:
you are right, I have consulted with the author and we will remove the review